lighthouse-cli/test/smokehouse/test-definitions/redirects-hash.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
smoke tests are slow, is it possible to combine this with an existing smoke test if it's just the hash we're after? It doesn't _need_ to use pushState, the requestedUrl could even include a hash and that wouldn't currently end up in `finalUrl`

Reply:
I did the client and server redirects test because the hash is *not* preserved over a client redirect (without adding it yourself) but it is preserved over a server redirect.